module/entities/helper.js

```diff
@@ -253,10 +253,10 @@ module.exports = class EntitiesHelper {
                 ) {
     
                     let getImmediateEntityTypes =
-                        await entityTypesHelper.list({
+                        await entityTypesHelper.entityTypesDocument({
```

Comment:
@aman-tunerlabs - Why do we have to change the name from the list to entityTypesDocument?